frontend/src/console/components/Hero/Hero.tsx

```diff
@@ -30,6 +30,7 @@ interface State {
 
   title: string;
   subtitle: string;
+  emptyFields: string;
```

Comment:
Use `boolean` instead of `string`.

Reply:
@rusrushal13 ^

Reply:
@giovannicimolin thanks, I have updated it :+1: